### DISCUSSION TITLE
add a googlemaps ifram inside a blog?

### DISCUSSION THREAD

**Discussion by jfvdvelden** [gethinode/hinode#604] [Help]
Hi to all,  
I upgrade to hinode from an other gohugo template.
Now i have several error i can't solve

formaly i had this code in my markdown blog:

<iframe src="https://www.google.com/maps/d/embed?mid=13QO3nHOnPjYGZYEEwQc4xFjHMACwmyg&ehbc=2E312F" width="640" height="480"></iframe>

but this is not working anymore and get this not working. Any help?

**Comment by markdumay** [comment]
Hinode uses strict security policies by default. You may need to adjust the [server headers ](https://gethinode.com/docs/advanced-settings/server-headers/)to allow the iframe to be loaded.
